src/App.js

```diff
@@ -1,29 +1,19 @@
 import React from 'react';
-
-import './App.css';
-
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import PostList from './Components/PostList';
+
+const preparedPostsWithUsersAndComments = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId),
+  comment: comments.filter(comment => comment.postId === post.id),
```

Comment:
```suggestion
  comments: comments.filter(comment => comment.postId === post.id),
```
У тебя тут будет 5 комментариев

Reply:
+